test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,34 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        #region Join tests
 
+        private static void AreElementsJoined(Element element, Element otherElement, bool expected)
+        {
+            bool result = element.IsJoined(otherElement);
+            Assert.AreEqual(expected, result);
+        }
+
+        [Test]
+        [TestModel(@".\joinedElements.rvt")]
+        public void CanSuccessfullyCheckIfTwoElementsAreJoined()
+        {
+            var wall1 = ElementSelector.ByElementId(184176, true);
```

Comment:
let's also document which elements are joined to which with `// comments` 
as a developer, i don't want to have to open the Revit file to check